trade-finance-manager-ui/server/controllers/case/index.test.js

```diff
@@ -673,7 +673,7 @@ describe('controllers - case', () => {
           allAmendments: expect.any(Array),
           amendmentsInProgress: expect.any(Array),
           amendments: expect.any(Array),
-          isTfmFacilityEndDateFeatureFlagEnabled: true,
+          shouldDisplayFacilityEndDate: false,
```

Comment:
how come this changed from `true` to `false`?

Reply:
The test in this file is using a non-GEF deal, so with the changes in this PR we'd now expect it to return false. The tests here are slightly odd in that they don't test lots of different cases and value combinations which is why I haven't added a test to make sure it's 'true' when it's a GEF facility, but possibly I should and break the current style of the file / test of that function